STEERING-COMMITTEE.md

```diff
@@ -162,11 +146,172 @@ There are two ways to raise issues to the steering committee for decision:
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
 
+## Composition
+
+The steering committee has 7 seats.  Five of the seats are Contributing
+seats, open to any project contributor.  Two of the seats are End-User
```

Comment:
Do we need to remove the end-user seats in this draft? It seems good in concept but it's been pretty clear from prior discussions that we don't have a readily available proven set of mechanics for doing this. Since the steering committee, within the context of this PR, could redefine their composition in the future, it seems like we could adopt the end-user seats later.

Reply:
Generally +1, though if we stick with 7 seats and try to add folks later, steering starts getting big.  What if we went with the 5 contributor seats (2 this cycle), with a goal to add 2 user seats later with staggered tenures?

Reply:
Either that, or you remove the end-user seats but add paragraphs to Eligibility encouraging end-users to run (and making sure they can meet the criteria).

The End-User seats were an addition by @bsnchan, can you step in here?

Reply:
I added these seats as an aspiration, to be filled in once the supporting mechanics are in place.